java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -73,6 +77,12 @@ class DatabaseImpl(
     DB_VERSION
 ) {
     private val mutex = Mutex()
+    // TODO: handle rehydrating from a snapshot.
+    private val stats = DatabasePerformanceStatistics(
+        insertUpdate = PerformanceStatistics(JvmTimer, *DatabaseCounters.INSERT_UPDATE_COUNTERS),
```

Comment:
FMI: what does the * do?

Reply:
It's a solar, turning an array into varargs